ruby/rb-kramdown/Portfile

```diff
@@ -1,21 +1,23 @@
-PortSystem		1.0
-PortGroup		ruby 1.0
-PortGroup		github 1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
You shouldn't make unrelated whitespace changes in the same commit as functional changes. That makes it harder for us to see your functional changes, both now as we review this PR and forever as we might need to look back on this commit in our history later.

It would be best in this PR to only make the functional changes that you desire. Then later you can submit another PR to make just the whitespace changes. That way it's easier for the maintainer of the portfile to, for example, accept the functional changes but reject the reformatting, if they wanted to do that.

Reply:
Ok, thanks.